### PR TITLE
Display Archived Collectives and Orgs Seperately in TopBar

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5770

image

Screen Shot 2022-12-02 at 05 56 38

### CODE REVIEW

File: components/TopBarProfileMenu.js [hunk 1]
```diff
@@ -155,16 +155,18 @@ class TopBarProfileMenu extends React.Component {
   }
 
   componentDidMount() {
-    document.addEventListener('keydown', this.handleKeyPress);
-    document.addEventListener('click', this.onClickOutside);
+    const main = document.querySelector('main');
+    main.addEventListener('keydown', this.handleKeyPress);
+    main.addEventListener('click', this.onClickOutside);
```
Comment (Betree) [inline components/TopBarProfileMenu.js:160]: Why?

**Review by Betree — APPROVED** [review]
Personally not a fan of having this button for all types, I would have added it only for Archived:
image

But could be useful, Idk. Maybe check with design to see what they think?

No blocker for me to merge overall, it's a great enhancement!

File: components/TopBarProfileMenu.js (hunk 1 above)
Comment (SudharakaP) [reply components/TopBarProfileMenu.js:157]: The reason to do this here is that note that targeting `document` means the menu will close whenever you click anywhere on the document; including within the menu itself. We don't want that; especially now that we have a drop-down within the menu when we click on the dropdown the menu shouldn't close. Ideally in general I don't think a menu should close when you click within the menu; only when the user clicks outside of the menu. To further illustrate this see the following screen-casts.

Before:

[before-dropdown.webm](https://user-images.githubusercontent.com/12435965/209723570-92c248c8-d74d-44a7-a761-10031c336cd2.webm)

After: 

[after-dropdown.webm](https://user-images.githubusercontent.com/12435965/209723645-194cd91a-3aca-4c95-a58f-0aeb7a43dea1.webm)

**Comment by SudharakaP** [comment]
> Personally not a fan of having this button for all types, I would have added it only for Archived: image
> 
> But could be useful, Idk. Maybe check with design to see what they think?
> 
> No blocker for me to merge overall, it's a great enhancement!

Yeah I think that's better. I've only added it for archived section. Will be merging soon. Thanks for the review 👍🏽

**Pull request merged**